benchmark.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
```

Comment:
Can you please add a `-h` flag that will echo the following?

```
# Usage:
# ./benchmark.sh [-ymn]
# -y flag skips the prompt
# -m flag runs maturin develop
# -n <number> sets the number of requests that oha sends
```

I feel like it will help us with legibility.